docs/administer/rbac.md

```diff
@@ -384,7 +384,7 @@ A new user in Percona Everest will initially have **no** permissions. To grant p
 
 
 ```sh
-kubectl edit configmap-rbac -n everest-system
+kubectl edit everest-rbac -n everest-system
```

Comment:
The correct command should be:
`kubectl edit cm everest-rbac -n everest-system`

Reply:
Done